pkg/providers/instance/instance.go

```diff
@@ -259,9 +264,7 @@ func getTags(ctx context.Context, nodeClass *v1beta1.NodeClass, nodeClaim *corev
 			v1alpha5.MachineManagedByAnnotationKey:                                         settings.FromContext(ctx).ClusterName,
 		}
 	} else {
-		overridableTags = map[string]string{
-			"Name": fmt.Sprintf("%s/%s", corev1beta1.NodePoolLabelKey, nodeClaim.Labels[corev1beta1.NodePoolLabelKey]),
-		}
+		overridableTags = map[string]string{}
```

Comment:
You can just remove this variable altogether if nothing is going to be in it anymore

Reply:
Didn't realize `lo.Assign` could take a `nil` map, looks like it is safe though so will do.